### ISSUE TITLE
Missing @types definition

### ISSUE THREAD

**Issue by olivierto** [rimiti/redis-keyman#37]
Hello, 

I'm using typescript for my API, and i'm surprised that Aclify doesn't provide @types definition files. 
This would be great ! 
I'm trying to get the first example `import {...} from "@aclify/aclify"` but with no success.... 

For info node acl from OptimalBits had one [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/acl/index.d.ts)

Cheers!

**Comment by rimiti** [comment]
Hello @olivierto,

Thank you for your feedback. 
I'm totally agree with you, it's a pain point of Aclify.

For your information, the package will be totally rewrited in Typescript soon. 🚀

**Comment by rimiti** [comment]
I'm closing this PR, the package has been rewritten.

**Comment by olivierto** [comment]
glad to hear and see that the plugin has been rewritten ! 
But it seems that the package still missing types definition file. 
```typescript
import * as Aclify from "@aclify/aclify";
```
Throws TS Error 
```
Could not find a declaration file for module '@aclify/aclify'. '/Users/me/Documents/sources/node_modules_custom/fastify-auth/node_modules/@aclify/aclify/dist/index.js' implicitly has an 'any' type.
      Try `npm install @types/aclify__aclify` if it exists or add a new declaration (.d.ts) file containing `declare module '@aclify/aclify';`

    8 import * as Aclify from "@aclify/aclify";
```

**Comment by olivierto** [comment]
created PR to fix this #47

**Comment by rimiti** [comment]
Thank you @olivierto for this PR. 🙏 
It has been merged.
